lib/vmpooler/pool_manager.rb

```diff
@@ -50,7 +50,10 @@ def check_pending_vm(vm, pool, timeout)
             rescue
             end
 
+            finish = '%.2f' % (Time.now - Time.parse($redis.hget('vmpooler__vm__' + vm, 'clone')))
```

Comment:
Addressed in https://github.com/sschneid/vmpooler/commit/18703efd60299de8a7949e654a059d88be13a0c2.